README.md

```diff
@@ -158,7 +160,7 @@ Repeat the steps above for all running NuoDB engine containers you want to monit
 
 ## Deploying NuoDB Collector using NuoDB Helm Charts
 
-NuoDB Collector will be deployed in sidecar containers using NuoDB Helm Charts.
+The NuoDB Collector is deployed in sidecar containers when deploying with the NuoDB Helm Charts.
```

Comment:
I dont think that this sentence is required at all here.

Reply:
Thanks, all the better. I was trying to rework this existing line assuming it was a desired detail to share, but I agree, seems like extra info not needed (keep it concise). 
I removed the line.